
syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.echo";
option java_outer_classname = "EchoProto";
option objc_class_prefix = "HLW";

package echo;

// The greeting service definition.
service Message {
  // Sends a message
  rpc SendMessage (SendMsg) returns (RecieveMsg) {}
}

// The  message to be send  
message SendMsg {
  string message = 1;
}

// The response message 
message RecieveMsg {
  string message = 1;
}
